src/app/course/_test/mock.ts

```diff
@@ -246,6 +247,20 @@ export const MOCK_MCQ_QUESTION: Question = {
     input_files: null,
 };
 
+export const MOCK_QUESTION_REPORT: QuestionReport = {
+    id: 1,
+    question_id: 1,
+    report: "TYPO_TEXT",
```

Comment:
change this and the rest when you change it to an enum

Reply:
Changed the report to use the new enum!